src/cli/UpdateCommand.cpp

```diff
@@ -104,8 +104,10 @@ sys.argv.append("{}")
     }
     cmd += fmt::format(R"python(
 import importlib.util
-spec = importlib.util.spec_from_file_location('__main__', r'{}')
+module_name = '__main__'
+spec = importlib.util.spec_from_file_location(module_name, r'{}')
 module = importlib.util.module_from_spec(spec)
+sys.modules[module_name] = module
```

Comment:
Do you have a link to somewhere where they do this `sys.modules["__main__"] = module` or something to convince me this is the right way to do it? Don't want to go chasing info if you already have it.

I have a _completely non-justified_ feeling this might cause trouble in some edge case we haven't foreseen, when this isn't a unittest file or something.

Reply:
You could be right. Here is my justification though.

https://docs.python.org/3/library/__main__.html#import-main

```
Python inserts an empty __main__ module in [sys.modules](https://docs.python.org/3/library/sys.html#sys.modules) at interpreter startup, and populates it by running top-level code....
```

Reply:
https://stackoverflow.com/a/62492464/2543372

Reply:
https://github.com/python/cpython/blob/754519a9f8c2bb06d85ff9b3e9fe6f967ac46d5c/Lib/unittest/loader.py#L336-L338